### ISSUE TITLE
Skipping current song via headphone button doesn't automatically play the next song

### ISSUE THREAD

**Issue by endervad** [jmshrv/finamp#107]
As in when you double-tap the button on headphones to skip the current song, you indeed skip the new song is paused. You have to manually start playing it via Play button in app or single-tapping headphone button.

**Comment by UnicornsOnLSD** [comment]
This also happens when pressing the skip button in the UI. I'm going to mess about with what happens with skipping so that the app does stuff like skip to the start of the song before going straight back. I'll also make sure to start playback after skipping if the player is paused.

**Comment by endervad** [comment]
Just letting you know that this bug still exists in 0.6.0.

**Comment by Chaphasilor** [comment]
Is this still happening @endervad? I didn't encounter this issue so far...

**Comment by jmshrv** [comment]
What device/headphones are you using?

**Comment by endervad** [comment]
Pixel 4a (non-5G) with Apple EarPods 3.5mm and Sony MDR-XB550. I've also tried clearing app's data just in case. And also this happens with both locked and unlocked screen if it matters.
On a side note: double-clicking doesn't register as "skip" while in the app, instead it's treated only as a single click. So playback just pauses and immediately resumes.

**Comment by Chaphasilor** [comment]
This is related to and blocked by https://github.com/ryanheise/just_audio/issues/901

**Comment by Chaphasilor** [comment]
closing in favor of #435
